<ui version="4.0" >
 <class>WdgLayerBox</class>
 <widget class="QWidget" name="WdgLayerBox" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>542</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="KisCmbComposite" native="1" name="cmbComposite" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Blending mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KoSliderCombo" native="1" name="doubleOpacity" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bnViewMode" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="KoDocumentSectionView" native="1" name="listLayers" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="bnAdd" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Create a new layer</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>image0</iconset>
       </property>
       <property name="popupMode" >
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bnLower" >
       <property name="minimumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Move layer down</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>image1</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bnRaise" >
       <property name="minimumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Move layer up</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>image2</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bnProperties" >
       <property name="minimumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>View or change the layer properties</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="bnDelete" >
       <property name="minimumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Delete the layer</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>image3</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KisCmbComposite</class>
   <extends></extends>
   <header>kis_cmb_composite.h</header>
  </customwidget>
  <customwidget>
   <class>KoDocumentSectionView</class>
   <extends></extends>
   <header location="global" >KoDocumentSectionView.h</header>
  </customwidget>
  <customwidget>
   <class>KoSliderCombo</class>
   <extends></extends>
   <header location="global" >KoSliderCombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
